.jenkins/pytorch/test.sh

```diff
@@ -278,12 +278,14 @@ test_libtorch() {
 
 test_vulkan() {
   if [[ "$BUILD_ENVIRONMENT" == *vulkan-linux* ]]; then
+    ln -sf "$TORCH_LIB_DIR"/libtorch* "$TORCH_TEST_DIR"
+    ln -sf "$TORCH_LIB_DIR"/libc10* "$TORCH_TEST_DIR"
```

Comment:
what's the source of getting to know which `lib*` to `ln` to the TEST_DIR?

Reply:
there maybe better approach but what I am doing now is reading ci failure logs

Reply:
I was wondering if we can simply just symlink everything?

Reply:
I thought about that but believe there should be a better way with cmake to let executable file looks for *.so files in the `lib` folder, so just leave to command here in this way to let it more trackable. Will create an issue to follow up soon